packages/ckeditor5-markdown-gfm/tests/_utils/utils.js

```diff
@@ -27,17 +27,17 @@ export function testDataProcessor( markdown, viewString, normalizedMarkdown, opt
 	if ( options && options.setup ) {
 		options.setup( dataProcessor );
 	}
-	const viewFragment = dataProcessor.toView( markdown );
 
+	const viewFragment = dataProcessor.toView( markdown );
 	const html = cleanHtml( _stringifyView( viewFragment ) );
 
 	// Check if view has correct data.
-	expect( html ).to.equal( viewString );
+	expect( JSON.stringify( html ) ).to.equal( JSON.stringify( viewString ) );
```

Comment:
This improves the readability when the output is different from the expected value.